test/scripts/data-preservers-run-hack.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
```

Comment:
Let's track the issue. So, once this hack is fixed by zombienet, we can remove this script.